lib-utilities/common/constants.go

```diff
@@ -218,8 +218,9 @@ var ResourceTypes = map[string]string{
 
 // Events contains the data with IP sent fro mplugin to PMB
 type Events struct {
-	IP      string `json:"ip"`
-	Request []byte `json:"request"`
+	IP        string `json:"ip"`
+	Request   []byte `json:"request"`
+	EventType string `json:"eventType"`
```

Comment:
Please define EventType as an enum with supported values.

Reply:
Marked as ToDo to change with plugin